pkg/forwarder/sync_forwarder.go

```diff
@@ -52,8 +54,9 @@ func (f *SyncForwarder) sendHTTPTransactions(transactions []*transaction.HTTPTra
 			// The intake may have closed the connection between Lambda invocations.
 			// If so, the first attempt will fail because the closed connection will still be cached.
 			log.Debug("Retrying transaction")
-			if err := t.Process(context.Background(), f.client); err != nil {
-				log.Warnf("SyncForwarder.sendHTTPTransactions failed to send: %s", err)
+			retryContext := context.WithValue(context.Background(), transaction.ContextKeyRetry, true)
+			if err := t.Process(retryContext, f.client); err != nil {
+				log.Errorf("SyncForwarder.sendHTTPTransactions failed to send: %s", err)
```

Comment:
Do we still want this to be a warning? Otherwise the invocation will be marked as an error. On the other hand if it's going to be very rare now, maybe it's better to make it an error.

Reply:
Yes I think a data loss would need to me marked as error, and since it should be extremely rare now let keep it as error?